build_tools/github_actions/tests/configure_ci_test.py

```diff
@@ -224,6 +224,7 @@ def test_main_windows_branch_push_matrix_generator(self):
                 "family": "gfx1151",
                 "pytorch-target": "gfx1151",
             },
+            {"family": "gfx120X-all", "pytorch-target": "gfx1201", "test-runs-on": ""},
```

Comment:
This is a change detector test (https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html) right now that isn't providing much added value, so we could rework the test.

Could instead check that the output of `matrix_generator()` has at least one item and all items match the expected format (`test-runs-on`, `family`, `pytorch-target`). Same for related test cases.

Where it could matter would be testing that `push` and `pull_request` triggers have their own changes to the matrix.